stubs/fpdf2/fpdf/enums.pyi

```diff
@@ -21,62 +19,62 @@ class CoerciveIntFlag(IntFlag):
     def coerce(cls, value: Self | str | int) -> Self: ...
 
 class WrapMode(CoerciveEnum):
-    WORD: str
-    CHAR: str
+    WORD = ...
```

Comment:
You may have answered this already elsewhere, in which case sorry, but how are type checkers supposed to be able to infer the type of `WrapMode.WORD.value` with something like this? I think mypy is currently able to infer a `str` here, but I don't think it would be able to with this change.

My preference would be to use the runtime values wherever possible (as you're already doing elsewhere in this PR) but, where it's not possible, to introduce linting rules that enforce annotations for this kind of pattern, e.g.

```suggestion
    WORD: str = ...
```

Reply:
Reading the discuss.python.org thread, I see you propose using `_value_: str` for cases where it's hard for us to provide the exact runtime value in the stub for each member. That will work in most cases, but it won't work for enums with heterogenous data types for their member values, e.g.

```py
class Foo(Enum):
    BAR = 99999999999999999999999999999999999999999999999999999999
    BAZ = "baaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaz"
```

You might say that that's an antipattern, and I might agree with you, but we should still have a way to express that in the stubs if that's what the runtime is doing. The most important thing in a stub is for us to be accurate w.r.t. to the runtime, even if the runtime is making use of a bad antipattern that we wouldn't necessarily endorse when we were writing our own code.

Reply:
@AlexWaygood, if you think that the `_value_` proposal is too constraining, please respond to the discuss.python.org thread and make your case there. Most of the participants in that broader thread are probably not following the detailed PR comments, so they won't see your above concerns.